### ISSUE TITLE
Social login buttons not showing up with adblock

### ISSUE THREAD

**Issue by niranjans** [meteor-useraccounts/bootstrap#22]
I just posted a question on [Stackoverflow](http://stackoverflow.com/questions/28613485/meteor-user-accounts-facebook-login-not-showing-but-google-is) and the answer showed a weird behavior:

> "Its a bit weird but it looks like while Meteor is unminified it blocks anything with facebook or twitter in its url such as the facebook/twitter packages' js code."

PS. I am not sure if this is the right place to report this.

**Comment by splendido** [comment]
Hi @niranjans!
...yes this is a good place, but pherhaps it would be better to open issues non specific to a particular fromnt-end in the [issue tracker for the core repo](https://github.com/meteor-useraccounts/core/issues).

This sounds weird to me (I'm developing under linux) because I used to get many different OAuth providers work without any problem!

...I've just read on SO that you somehow solved it: is this correct?
Which browser are you testing it on?

We might report this problem within the documentation as a known bug in case we can confirm it.

Let me know!

**Comment by niranjans** [comment]
Hi @splendido 

Yes, the problem was solved. It was just a very trivial solution - Disabling Adblock Plus for localhost. Everything works fine once that is disabled. This had me stumped for hours though. 

Initially I thought this was a system specific problem but I realized this problem is only on my Chrome browser with the AdBLock Plus plugin (on Linux Mint 17). 

Maybe UserAccounts is pulling something that is in the block list? But again, this does not happen on other browsers or even chrome on a Mac system. 

Thanks, 
Niranjan
